test/Extension.Tests.ps1

```diff
@@ -1,6 +1,6 @@
 Describe 'SecretManagement.LastPass tests' {
     BeforeAll {
-        & $PSScriptRoot/reload.ps1
+        . $PSScriptRoot/reload.ps1
```

Comment:
Since I defined a `ConvertFrom-SecureString` function in Reload.ps1, it need to be dot-sourced so everything is in the current scope.